src/client/components/app/LineDetailsDisplay/PositionsTable.tsx

```diff
@@ -177,6 +178,15 @@ export const PositionsTable = (props: PositionsProps) => {
     dispatch(ModalsActions.openModal({ modalName: 'addPosition' }));
   };
 
+  const isWithdrawable = (deposit: string, borrowed: string, lender: string) => {
+    if (!userWallet) {
+      return;
+    }
+    return (
+      Number(borrowed) < Number(deposit) && ethers.utils.getAddress(lender) === ethers.utils.getAddress(userWallet!)
```

Comment:
@kibagateaux what is your preferred method for comparing addresses?

Reply:
That's what I am doing here, no?